scripts/src/models/users.ts

```diff
@@ -21,7 +21,8 @@ export class User extends CreationDateModel {
 	public activatedDate?: Date;
 
 	public get activated(): boolean {
-		return !!this.activatedDate;
+		return true; // XXX for now all are activated
```

Comment:
this on purpose or for local testing?

Reply:
He's disabling activation because it created problems for wloo, see his convo with Noa

Reply:
yes